docs/spelling_wordlist.txt

```diff
@@ -73,6 +75,7 @@ plugin
 plugins
 pluggable
 pre
+preceding
```

Comment:
At least locally this is a recognised word which does not need to be added to the dictionary.